typings/glob/glob.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for Glob 5.0.10
```

Comment:
checking in the typings can be a valid thing to do, because the github connection can be flaky...
but in our other projects, we run `typings install ...` in an npm postinstall script and don't check in the files